docs/docs/guides/07_dapps/lightweight-dapp.md

```diff
@@ -0,0 +1,275 @@
+---
+sidebar_position: 4
+sidebar_label: 'Tutorial: Lightweight dApp Development'
+---
+
+# Lightweight dApp Development
+
+In web development, the term "bundle size" refers to the aggregate size of all the files that are delivered to a user when they visit a web page. Large bundle sizes result in long loading times, poor performance, and overall bad user experience. A web application with a relatively small bundle size can be described as "lightweight".
+
+The [modular design](/#packages--plugins) of Web3.js promotes lightweight dApp development by allowing developers to include only the capabilities that are required for their dApp. Furthermore, Web3.js makes use of modern [JavaScript module](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) syntax to facilitate [tree shaking](https://developer.mozilla.org/en-US/docs/Glossary/Tree_shaking), which is a powerful mechanism for removing unused code that would otherwise be included in a web application.
+
+This guide will apply lightweight dApp development techniques to the minimal React template provided by the [`create-web3js-dapp` command-line utility](https://github.com/web3/create-web3js-dapp). [Webpack Bundle Analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer) will be used to analyze the impact of these techniques.
+
+## Overview
+
+Here is a high-level overview of the steps in this tutorial:
+
+1. Review prerequisites
+2. Initialize a new Web3.js dApp and add dependencies
+3. Configure and analyze a development build
+4. Analyze the built-in React production build
+5. Use only necessary packages and analyze the result
+
+:::tip
+If you encounter any issues while following this guide or have any questions, don't hesitate to seek assistance. Our friendly community is ready to help you out! Join our [Discord](https://discord.gg/F4NUfaCC) server and head to the **#web3js-general** channel to connect with other developers and get the support you need.
```

Comment:
https://discord.gg/F4NUfaCC is not valid.

Reply:
Good catch 🙏🏻 I've updated this in all the guides/tutorials.